test/compilable/testDIP37.d

```diff
@@ -30,7 +30,6 @@ void test7()
     static import pkgDIP37.datetime;
     static assert(!__traits(compiles, def()));
     pkgDIP37.datetime.def();
-    pkgDIP37.datetime.common.def();
```

Comment:
Rather than deleting this, should we instead add `static import pkgDIP32.datetime.common`, perhaps in a different scope?

Reply:
I don't see the point in that: the code will error if that different scope does not include `test7`'s scope.

Reply:
The alternative would be `static assert(!__traits(compiles, ...)`